jazz_apigee-proxy-aws/components/logger.js

```diff
@@ -0,0 +1,141 @@
+// =========================================================================
```

Comment:
Can we use an off-the-shelf simple logger rather than rolling our own?

Reply:
I agree although this is part of our template that is being used in all the platform services. We will have to discuss this on what logger should we use as this is a bigger change.

Reply:
Holy heck you're right, we're copypasting the same hand-rolled logger dozens of times throughout the codebase, it's worse than I thought.

Reply:
There are other things like that we need to improve and we are slowly working towards that.

Reply:
Agreed. Don't remember why we ended up using custom logger instead of standard logger like log4js. Time to create an issue to update the core services and make sure that the logging is functional end to end (we parse them in log streamer).

Reply:
@devsatishm I can create a task to use an off-the-shelf Node logger.